extensions/common/vault/hashicorp-vault/src/main/java/org/eclipse/dataspaceconnector/core/security/hashicorpvault/HashicorpVaultExtension.java

```diff
@@ -63,16 +65,24 @@ public PrivateKeyResolver privateKeyResolver() {
         return privateKeyResolver;
     }
 
+    @Provider
```

Comment:
Do we still need to diverge here from the EDC implementation? I thought we could reuse their code here since they also added Certificate Resolver in their code: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/9bec10ce09ef36d85156fe897b946ac88be6e330/extensions/common/vault/hashicorp-vault/src/main/java/org/eclipse/dataspaceconnector/core/security/hashicorpvault/HashicorpVaultExtension.java

Reply:
good point, thank you - I will check